templates/CRM/Contact/Page/View/Summary.tpl

```diff
@@ -116,9 +116,14 @@
       </ul>
 
       {foreach from=$allTabs item=tabValue}
-        {if !empty($tabValue.template)}
+        {if $tabValue.template}
```

Comment:
Might this cause an undefined index warning? I think you're right we're not supposed to use empty() in templates but maybe we should use isset() to prevent the e-notice.

Reply:
I changed to make consistent with https://github.com/civicrm/civicrm-core/pull/31606/files#r1888888349

Reply:
but maybe should have changed the other way or changed both to isset?